xml/art_sle_amazon_ecs.xml

```diff
@@ -21,9 +21,9 @@
   <?dbtimestamp format="B d, Y" ?></date>
  <abstract>
    <para>
-    Amazon EC2 Container Service (ECS) is a container management service
-    that provides a set of APIs for scheduling container workloads
-    across EC2 clusters.
+    <link xlink:href="https://aws.amazon.com/ecs/">Amazon EC2 Container Service (ECS)</link>
+    is a container management service that provides a set of APIs for
+    scheduling container workloads across EC2 clusters.
    </para>
    <para>
     SUSE Linux Enterprise 12 images with the
```

Comment:
Replace "SUSE Linux Enterprise" with the entity `&sle;`.

You can find additional entity definitions in the file `entity-decl.ent`.